packages/jest-parallel/README.md

```diff
@@ -0,0 +1,173 @@
+# jest-parallel
+
+Module for executing heavy tasks under forked processes in parallel, by providing a modern interface (`Promise` based), minimum overhead, and sticky workers.
+
+The module works by providing an absolute path of the module to be loaded in all forked processes, as well as the list of methods that can be remotely called. All methods are exposed on the parent process as promises, so they can be `await`'ed. Child (worker) methods can either be synchronous or asynchronous.
+
+The way sticky workers work is by using the returned string of the `getWorkerKey` method. If the string was used before, the call will be queued to the related worker; if not, it will be executed by the first available worker, then sticked to the worker that executed it; so the next time it will be processed by the same worker. If you have no preference on the worker executing the task, but you have defined a `getWorkerKey` method because you want _some_ of the tasks to be sticked, you can return `null` from it.
+
+
+## Install
+
+```sh
+$ npm install --save jest-parallel
+```
+
+
+## API
+
+The only exposed method is a constructor (`JestFarm`) that is initialized by passing an options object. This object can have the following properties:
+
+
+### `exposedMethods: $ReadOnlyArray<string>` (required)
+
+List of method names that can be called on the child processes from the parent process.
+
+
+### `workerPath: string` (required)
+
+Absolute path of the file loaded in child processes. Use `require.resolve` to transform a relative path into an absolute one.
+
+
+### `workers: number` (required)
+
+Amount of workers to spwan.
+
+
+### `forkOptions: Object` (optional)
+
+Allow customizing all options passed to `childProcess.fork`. By default, some values are set (`cwd` and `env`), but you can override them and customize the rest. For a list of valid values, check [the Node documentation](https://nodejs.org/api/child_process.html#child_process_child_process_fork_modulepath_args_options).
+
+
+### `getWorkerKey: (method: string, ...args: Array<any>) => ?string` (optional)
```

Comment:
I would go with @davidaurelio's suggestion and use `computeWorkerKey` because this function is potentially expensive, and it would be good to denote that in the name.